examples/reentrancy_guard/flipper/Cargo.toml

```diff
@@ -41,5 +43,9 @@ std = [
     "scale-info/std",
 
     "brush/std",
+    "reentrancy_guard/std",
+]
+
+ink-as-dependency = [
+    "reentrancy_guard/ink-as-dependency"
```

Comment:
btw, why do you need to import brush contracts with ink-as-dependency?

Reply:
If someone imports Flipper as ink-as-dependency, all sub-imports also must be ink-as-dependency to avoid unexpected errors.